packages/ui/src/composables/useEmitProxy.ts

```diff
@@ -11,7 +11,7 @@ export const useEmitProxy = <T extends string>(events: T[]) => {
   const createListeners = (emit: (event: T, ...args: any[]) => void) => {
     return events.reduce((acc, key) => ({
       ...acc, [eventToListenerName(key)]: (...args: any[]) => emit(key, ...args),
-    }), {} as Record<T, any>)
+    }), {} as Record<string, any>)
```

Comment:
Looks like Maksim will not like this :D

Reply:
So actually Record can't contain  `T` type since here 
```javascript
[eventToListenerName(key)]
```
it changes, example: 
 ```javascript
const event = 'focus'
cons result = eventToListenerName(event)
console.log(result) // onFocus
```
As we could see, output values are not the same as input values